stdlib/hashtbl.ml

```diff
@@ -430,22 +444,41 @@ module MakeSeeded(H: SeededHashedType): (SeededS with type key = H.t) =
           else find_in_bucket next in
       find_in_bucket h.data.(key_index h key)
 
-    let rec replace_bucket key data = function
-      | Empty ->
-          true
-      | Cons ({key=k; next} as slot) ->
-          if H.equal k key
-          then (slot.key <- key; slot.data <- data; false)
-          else replace_bucket key data next
+      let rec retrieve_bucket key bucket =
```

Comment:
Everything has been indented by two spaces here. Please dedent back to the original.